### ISSUE TITLE
Is --recurse mode on by default?

### ISSUE THREAD

**Issue by midenok** [aspiers/git-deps#90]
I run
```
git-deps HEAD
70054a7a01000af478b0e790a2935df2042da75d 232ca5f1a57500d78416d8e830dd7480343af732
70054a7a01000af478b0e790a2935df2042da75d 0c571f8c4ee6235327253d5eba58a63f03f12627
70054a7a01000af478b0e790a2935df2042da75d 82114170bcfad83408723dd6876e4e1a5e8c7148
70054a7a01000af478b0e790a2935df2042da75d a9a56b235543fd34ba57c27624aa269c0a13df9d
70054a7a01000af478b0e790a2935df2042da75d 2dbeebdb16436e3c2723cd483aaf21d93de799d6
70054a7a01000af478b0e790a2935df2042da75d f9b8c908a0abbee756f2ef0b100dd9d173bc3e97
70054a7a01000af478b0e790a2935df2042da75d 2a33d248e0bd910ec10a2bb68e47f17b47e3a980
a92663ea5341053f2645a7fd5beba802b67f5372 d2408e43f912d91de3c982330d62f717e63efbc5
a92663ea5341053f2645a7fd5beba802b67f5372 64a60185944131d985c224d3e4d3a879c1ca3296
a92663ea5341053f2645a7fd5beba802b67f5372 0f9ddfa9d8bb8d071266bcc63e92813cf18ccd2b
```
And get two columns of output which according to documentation corresponds to `--recurse` mode. The command continues endlessly with no means to stop (^C doesn't work, I exit via ^Z, kill %1). How to break it? How to specify maximum depth?

**Comment by aspiers** [comment]
`--recurse` is not on by default, but when `HEAD` is the argument then you are doing dependency analysis on the entire history, not just `HEAD`.  The way to do the latter is:

    git deps HEAD^!

Since you already found this in #82, and there is #67 filed to improve the UX, I'll close this.

**Comment by midenok** [comment]
Please take into account it was not user-friendly: ^C didn't work.

**Comment by aspiers** [comment]
I did: fixed ^C in #94.
